cetmix_tower_server/tests/test_command.py

```diff
@@ -41,6 +74,59 @@ def setUp(self, *args, **kwargs):
     COMMAND_RESULT = {"exit_code": 0, "message": "New command was created"}
 else:
     COMMAND_RESULT = {"exit_code": -1, "message": "error"}
+    """,
+            }
+        )
+
+        self.command_create_python_command_1 = self.Command.create(
```

Comment:
Just wondering why `command_create_python_command_1` with `create`?
Why not simply `command_python_command_1`? Is there any specific meaning behind `create` here?
Same question for all other commands.

Reply:
I'll fix it